source/guides/network/connecting-to-mongodb.rst

```diff
@@ -0,0 +1,124 @@
+.. post:: 13 Feb, 2020
+    :category: Network
+    :excerpt: 1
+    :nocomments:
+
+#########################
+How to connect to MongoDB
+#########################
+
+Learn how to access your API node MongoDB instance.
+
+**********
+Background
+**********
+
+The :doc:`REST Gateway <../../api>` offers a broad range of endpoints so that you don't have to connect to MongoDB. Still, if you are developing new plugins for Symbol, or you are analyzing extensive blockchain data, you might be looking to connect MongoDB directly.
+
+By the end of this guide, you will be connected to your API node database instance and doing some basic queries.
+
+*************
+Prerequisites
+*************
+
+- Finish the :doc:`getting started section <../../getting-started/setup-workstation>`
+- Have an :doc:`API or Dual node running <running-a-test-net-node>`
+
+.. _install-robo3t:
+
+***************
+Install Robo 3T
+***************
+
+For this tutorial, we are going to use |Robo3T| (formerly RoboMongo), a cross-platform MongoDB management tool, to interact with the database.
+
+1. Download Robo 3T `here <https://robomongo.org/download>`_.
+2. Open the installer and follow the installation instructions.
+
+In case of doubt, follow the `official installation docs <https://studio3t.com/knowledge-base/articles/installation/>`_.
+
+***********************
+Create a new connection
+***********************
+
+1. Open a new terminal in the computer you are running the node. Then, get the **container's identifier** running MongoDB with ``docker ps``.
```

Comment:
1. Open a new terminal on the computer you are running the node. Then, get the **container's identifier** running MongoDB with ``docker ps``.